pkg/services/grafana-apiserver/rest/dualwriter.go

```diff
@@ -67,10 +70,17 @@ func NewDualWriter(legacy LegacyStorage, storage Storage) *DualWriter {
 // Create overrides the default behavior of the Storage and writes to both the LegacyStorage and Storage.
 func (d *DualWriter) Create(ctx context.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
 	if legacy, ok := d.legacy.(rest.Creater); ok {
-		_, err := legacy.Create(ctx, obj, createValidation, options)
+		created, err := legacy.Create(ctx, obj, createValidation, options)
 		if err != nil {
 			return nil, err
 		}
+		obj = created // write the updated version
+		rsp, err := d.Storage.Create(ctx, obj, createValidation, options)
+		if err != nil {
+			// TODO
```

Comment:
@toddtreece -- thoughts on what to do here?  For now just a comment?

The next big step is a reconciliation pattern -- something that will make sure the legacy values are in sync with storage.  Maybe that is our fallback answer on how error cascading will work